site/content/en/docs/tasks/run_rayjobs.md

```diff
@@ -106,83 +106,7 @@ data:
-kind: RayJob
-metadata:
-  generateName: ray-job-sample-
-  labels:
-    kueue.x-k8s.io/queue-name: user-queue
-spec:
-  shutdownAfterJobFinishes: true
-  entrypoint: python /home/ray/samples/sample_code.py
-  runtimeEnv: ewogICAgInBpcCI6IFsKICAgICAgICAicmVxdWVzdHM9PTIuMjYuMCIsCiAgICAgICAgInBlbmR1bHVtPT0yLjEuMiIKICAgIF0sCiAgICAiZW52X3ZhcnMiOiB7ImNvdW50ZXJfbmFtZSI6ICJ0ZXN0X2NvdW50ZXIifQp9Cg==
-  rayClusterSpec:
-    rayVersion: '2.4.0' # should match the Ray version in the image of the containers
-    # Ray head pod template
-    headGroupSpec:
-      # the following params are used to complete the ray start: ray start --head --block --redis-port=6379 ...
-      rayStartParams:
-        dashboard-host: '0.0.0.0'
-        num-cpus: '1' # can be auto-completed from the limits
-      #pod template
-      template:
-        spec:
-          containers:
-            - name: ray-head
-              image: rayproject/ray:2.4.0
-              ports:
-                - containerPort: 6379
-                  name: gcs-server
-                - containerPort: 8265 # Ray dashboard
-                  name: dashboard
-                - containerPort: 10001
-                  name: client
-                - containerPort: 8000
-                  name: serve
-              resources:
-                limits:
-                  cpu: "2"
-                requests:
-                  cpu: "1"
-              volumeMounts:
-                - mountPath: /home/ray/samples
-                  name: code-sample
-          volumes:
-            # You set volumes at the Pod level, then mount them into containers inside that Pod
-            - name: code-sample
-              configMap:
-                # Provide the name of the ConfigMap you want to mount.
-                name: ray-job-code-sample
-                # An array of keys from the ConfigMap to create as files
-                items:
-                  - key: sample_code.py
-                    path: sample_code.py
-    workerGroupSpecs:
-      # the pod replicas in this group typed worker
-      - replicas: 3
-        minReplicas: 1
-        maxReplicas: 5
-        # logical group name, for this called small-group, also can be functional
-        groupName: small-group
-        rayStartParams: {}
-        #pod template
-        template:
-          spec:
-            containers:
-              - name: ray-worker # must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc'
-                image: rayproject/ray:2.4.0
-                lifecycle:
-                  preStop:
-                    exec:
-                      command: [ "/bin/sh","-c","ray stop" ]
-                resources:
-                  limits:
-                    cpu: "2"
-                  requests:
-                    cpu: "1"
-```
+{{% include "jobs/ray-job-sample.yaml" "yaml" %}}
 
 You can run this RayJob with the following commands:
```

Comment:
The included sample contains both the config map and the sample. It seems the config map should be removed from the included file, as it is in a separate dedicated file. Also, the instructions below suggest the config map is meant to be created separately.

Reply:
Thanks for your suggestion, I have modified it